travis-karma.conf.js

```diff
@@ -0,0 +1,58 @@
+module.exports = function(config) {
+  // Browsers to run on Sauce Labs
+  var customLaunchers = {
+    SauceLabsChrome: {
+      base: 'SauceLabs',
+      browserName: 'chrome',
+      version: '68.0',
+      platform: 'OS X 10.12'
+    },
+    SauceLabsFirefox: {
+      base: 'SauceLabs',
+      browserName: 'firefox',
+      version: '64.0',
+      platform: 'Windows 10'
+    }
+  }
```

Comment:
Can we pin  the browser to latest(https://wiki.saucelabs.com/display/DOCS/Test+Configuration+Options)(that way as browsers update our tests are testing on those new browsers and letting us knew when the browsers broke something without us having to manually update these versions), and if this works can you update it to test both chrome & Firefox on windows & os x?

Reply:
Yup, I don't see why not